recipes/fc/build.sh

```
@@ -0,0 +1,8 @@
+
+#!/bin/bash
```

Shellcheck

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)